docs/developers/strat-lib/guides/HowToTest.md

```diff
@@ -92,7 +92,7 @@ Let us unfold the specification for the test itself.
 For this first test, we want to 
 
 * post and fund a pair of offers using [`Amplifier.newAmplifiedOffers(...)`](./DirectHowTo.md#publishing-amplified-liquidity), 
-* [snipe](../../protocol/technical-references/taking-and-making-offers/taker-order/README.md#offer-sniping) *one* of the offers, and,
+* snipe *one* of the offers, and,
```

Comment:
Sniping does not exist anymore

Reply:
Indeed. However, this PR does not update the strat lib section, only the protocol, so for now I just removed the broken links